crates/common_enums/src/enums.rs

```diff
@@ -342,6 +342,7 @@ pub enum ConnectorType {
     PaymentMethodAuth,
 }
 
+///The three letter ISO currency code in uppercase. Eg: 'USD' to charge US Dollars
```

Comment:
`charging` is context sensitive. We use this enum for a variety of purposes.
```suggestion
/// The three letter ISO currency code in uppercase. Eg: 'USD' for the United States Dollar.
```